src/WebJobs.Script.WebHost/Management/FunctionsSyncManager.cs

```diff
@@ -366,6 +378,35 @@ public async Task<SyncTriggersPayload> GetSyncTriggersPayload()
             };
         }
 
+        internal async Task<JObject> GetHostJsonExtensionsAsync()
+        {
+            var hostOptions = _applicationHostOptions.CurrentValue.ToHostOptions();
+            string hostJsonPath = Path.Combine(hostOptions.RootScriptPath, ScriptConstants.HostMetadataFileName);
+            if (FileUtility.FileExists(hostJsonPath))
+            {
+                try
+                {
+                    _logger.LogInformation("Reading host.json for SyncTrigger.");
```

Comment:
Please remove this log statement - it'll just be noise.

Reply:
If this needed for debugging - please change to Trace level

Reply:
I'd argue that its not needed at all. Note that on the other existing path for Durable where we're reading host.json, we don't log anything. I'd just remove it :)

Reply:
Ok, I removed it.